servers/physics/joints/generic_6dof_joint_sw.cpp

```diff
@@ -221,7 +221,7 @@ Generic6DOFJointSW::Generic6DOFJointSW(BodySW *rbA, BodySW *rbB, const Transform
 void Generic6DOFJointSW::calculateAngleInfo() {
 	Basis relative_frame = m_calculatedTransformA.basis.inverse() * m_calculatedTransformB.basis;
 
-	m_calculatedAxisAngleDiff = relative_frame.get_euler();
+	m_calculatedAxisAngleDiff = -relative_frame.get_euler();
```

Comment:
Don't ask me way. It works this way, convention must have changed for the computed matrix